deeptools/computeMatrix.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+import argparse
+import sys
+import os
+import os.path
+import shutil
+
+from deeptools.parserCommon import writableFile, numberOfProcessors
+from deeptools import heatmapper
+from deeptools._version import __version__
+import deeptools.config as cfg
+import deeptools.utilities
+
+def parse_arguments(args=None):
+    parser = \
+        argparse.ArgumentParser(
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description="""
+
+This tool summarizes and prepares an intermediary file containing
+scores associated with genomic regions that can be used afterwards to
+plot a heatmap or a profile.  Typically, these genoic regions are
+genes, but any other regions defined in a BED format can be
+used. This tool can also be used to filter and sort regions according
+to their score.
+
+To learn more about the specific parameters type:
+
+%(prog)s reference-point --help or
+%(prog)s scale-regions --help
+
+""",
+            epilog='An example usage is:\n  %(prog)s reference-point -S '
+            '<biwig file> -R <bed file> -b 1000\n \n')
+
+    parser.add_argument('--version', action='version',
```

Comment:
I do not clearly understand... the background is, I tried to keep each tool with its own documentation, except when the documentation is shared by other tools in which case it is located in deeptools/parserCommon.py